homeassistant/components/alarm_control_panel/alarmdotcom.py

```diff
@@ -0,0 +1,97 @@
+"""
+homeassistant.components.alarm_control_panel.alarmdotcom
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Interfaces with Verisure alarm control panel.
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/alarmdotcom/
+"""
+import logging
+
+import homeassistant.components.alarm_control_panel as alarm
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+
+from homeassistant.const import (
+    STATE_UNKNOWN,
+    STATE_ALARM_DISARMED, STATE_ALARM_ARMED_HOME, STATE_ALARM_ARMED_AWAY)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+REQUIREMENTS = ['https://github.com/Xorso/pyalarmdotcom'
+                '/archive/0.0.7.zip'
+                '#pyalarmdotcom==0.0.7']
+DEFAULT_NAME = 'Alarm.com'
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Setup Alarm.com control panel """
+
+    username = config.get(CONF_USERNAME)
+    password = config.get(CONF_PASSWORD)
+
+    if username is None or password is None:
+        _LOGGER.error('Must specify username and password!')
+        return False
+
+    add_devices([AlarmDotCom(hass,
+                             config.get('name', DEFAULT_NAME),
+                             config.get('code'),
```

Comment:
Just a heads up, numbers in the config will be returned as an `int` or `float`. If you expect a `string`, make sure you convert it.